Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Use latest stable (LTS) alpine releases
+
+# Stage 1: Build container
+FROM node:lts-alpine as builder
+
+# Install dependencies
+# Git and python are currently necessary to install some packages
+RUN apk update
```

Comment:
Can you also inline these apk-commands into oneliner and clean the caches after? Having smaller images and less layers to cache is friendlier to the users :slightly_smiling_face:

Reply:
ok